src/__test__/redux/reducers/__snapshots__/experimentSettingsReducer.test.js.snap

```diff
@@ -102,7 +102,122 @@ Object {
     "experimentId": null,
     "experimentName": null,
   },
-  "originalProcessing": Object {},
+  "originalProcessing": Object {
```

Comment:
This is added to the snapshot because the function `generateMockProcessingConfig` now also adds values into `originalProcessingConfig`. This is the same as what currently happens in the reducer, where loaded experiment settings are copied into `originalProcessing` settings.